### PR TITLE
Remove nordderper restart

### PR DESCRIPTION
**Pull request by gytsto**
### Problem
*--describe problem being solved--*

### Solution
*--describe selected solution--*

### :ballot_box_with_check: Definition of Done checklist
- [x] Commit history is clean ([requirements](../../blob/main/docs/git_commit_messages_requirements.md))
- [x] README.md is updated
- [x] changelog.md is updated

### CODE REVIEW

File: nat-lab/tests/mesh_api.py [hunk 1]
```diff
@@ -273,9 +275,14 @@ def config_dynamic_nodes(self, node_configs: List[bool]) -> Tuple[Node, ...]:
                 public_key=str(public),
                 is_local=is_local,
             )
-            # TODO correct subnet when we'll decide about the range
-            self.assign_ip(node.id, f"100.64.33.{node_idx}")
-            self.assign_ip(node.id, f"fd00::dead:{node_idx}")
+            ipv4 = (
+                f"100.{random.randint(65, 127)}.{random.randint(0, 255)}.{random.randint(8, 255)}"
```
Comment (matislovas) [inline nat-lab/tests/mesh_api.py:279]: Not including `64` in second octet, and last octet should exclude `255` which is broadcast, but these are nitts/edge cases.

File: nat-lab/tests/mesh_api.py [hunk 2]
```diff
@@ -273,9 +275,14 @@ def config_dynamic_nodes(self, node_configs: List[bool]) -> Tuple[Node, ...]:
...
+            )
+            ipv6 = (
+                f"fc74:656c:696f:0:{format(random.randint(0, 0xFFFF), 'x')}:{format(random.randint(0, 0xFFFF), 'x')}:{format(random.randint(0, 0xFFFF), 'x')}:{format(random.randint(0, 0xFFFF), 'x')}"
```
Comment (matislovas) [inline nat-lab/tests/mesh_api.py:282]: `:0:` can be changed to `::` , last segment should have range of  `(8, 0xFFFF)` to account for reserved addresses.

**Comment by Hasan6979** [comment]
lgtm! +1

File: nat-lab/tests/mesh_api.py (hunk 1 above)
Comment (gytsto) [reply nat-lab/tests/mesh_api.py]: updated

File: nat-lab/tests/mesh_api.py (hunk 2 above)
Comment (gytsto) [reply nat-lab/tests/mesh_api.py]: updated, but left `:0:` since there is no proper ipv6 string comparison... will leave it for future implementation

**Comment by matislovas** [comment]
+0.5 for corrected IPv6 ranges

**Review by Jauler — APPROVED** [review]
+2.0
